docs/resources/confluent_schema_exporter.md

```diff
@@ -98,6 +129,8 @@ The following arguments are supported:
 - `status` - (Optional String) The status of the schema exporter. Accepted values are: `RUNNING` and `PAUSED`.
 - `reset_on_update` - (Optional Boolean) The flag to control whether to reset the exporter when updating configs. Defaults to `false`.
 
+-> **Note:** When using OAuth authentication in the provider block, `destination_schema_registry_cluster.id` is required and credentials blocks for both source and destination schema registry clusters must be removed.
```

Comment:
The verification doc implies a user needs to patch TF state file and not just TF configuration file, is it accurate? If yes, we may want to provide additional instructions here.